<launch>
    <!-- ARDUINO -->
    <!--  * encoders and ultrasound sensors -->
    <node pkg="rosserial_python" type="serial_node.py" name="arduino_node" >
        <param name="port" value="/dev/ttyUSB0" />
    </node>


    <!-- IMU -->
    <node pkg="razor_imu_9dof" type="imu_node.py" name="imu_node">
        <param name="port" value="/dev/ttyACM0" />
        <param name="baud_rate" value="115200" />
    </node>


    
    <!-- RECORD DATA TO DATOR -->
    <node pkg="barc" type="record_experiment.py" name="record_experiment" output="screen">
        <param name="experiment_name" value="rc-record-test-zeno-01" />
        <param name="camera_on" value="false" />
    </node>

    <!-- MANUAL RC JOYSTICK CONTROL -->
    <node pkg="barc" type="controller_rc.py" name="manual" output="screen" />

</launch>
